presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -240,7 +241,8 @@ public JdbcTableHandle getTableHandle(JdbcIdentity identity, SchemaTableName sch
                     if (columnMapping.isPresent()) {
                         String columnName = resultSet.getString("COLUMN_NAME");
                         boolean nullable = columnNullable == resultSet.getInt("NULLABLE");
-                        columns.add(new JdbcColumnHandle(connectorId, columnName, typeHandle, columnMapping.get().getType(), nullable));
+                        String comment = resultSet.getString("REMARKS");
+                        columns.add(new JdbcColumnHandle(connectorId, columnName, typeHandle, columnMapping.get().getType(), nullable, emptyToNull(comment)));
```

Comment:
Can you explain the use of emptyToNull?

Reply:
Mysql driver returns empty in the case of null comments with the columns.

Reply:
Can we make this more specific to MySQL then? It seems like the distinction between empty and null shouldn't get lost for other JDBC data sources. It's fine if that is a follow up or a starter task you give to someone else.